arduino/smartcar/smartcar.ino

```diff
@@ -472,6 +483,35 @@ if (ifLocalhost){
      }
 }
 
+void startCamera()
+{
+#ifdef __SMCE__
+    Camera.begin(QVGA, RGB888, 15);
+    frameBuffer.resize(Camera.width() * Camera.height() * Camera.bytesPerPixel()); //setup
+#endif
+}
+
+// Method to publish Camera Data
+//example:
+// CameraData (true, "/smartcar/camera/front" , front); // ex how to use in loop method
+void cameraData(boolean pubCameraData)
```

Comment:
The example in line 496 has different parameters from the method, but I'm not sure, maybe they're referring to different things?

Reply:
Exactly, that could have been used if you consider designing your method without boolean if-statement.